packages/devtools_app/lib/src/shared/config_specific/framework_initialize/_framework_initialize_web.dart

```diff
@@ -83,14 +81,18 @@ void _sendKeyPressToParent(KeyboardEvent event) {
   // will need to be posted up to the parent
   // https://github.com/flutter/devtools/issues/2775
 
-  // Check we have a connection and we appear to be embedded somewhere expected
-  // because we can't use targetOrigin in postMessage as only the scheme is fixed
-  // for VS Code (vscode-webview://[some guid]).
-  if (globals.containsKey(ServiceConnectionManager) &&
-      !serviceConnection.serviceManager.connectedState.value.connected) {
```

Comment:
I remove this check for connection because we need to support this for things like the Property Editor that has no VM Service. I believe the original thought behind this check was just to minimize the places in which we would forward on events. To restrict this a little more, I added "Code/" to the user agent check below so we'll definitely only trigger in VS Code and on Desktop.